operator/.bingo/variables.env

```diff
@@ -12,11 +12,13 @@ BINGO="${GOBIN}/bingo-v0.7.0"
 
 CONTROLLER_GEN="${GOBIN}/controller-gen-v0.10.0"
 
+GEN_CRD_API_REFERENCE_DOCS="${GOBIN}/gen-crd-api-reference-docs-v0.4.2"
+
 GOFUMPT="${GOBIN}/gofumpt-v0.4.0"
 
 GOLANGCI_LINT="${GOBIN}/golangci-lint-v1.50.0"
 
-HUGO="${GOBIN}/hugo-v0.104.3"
+HUGO="${GOBIN}/hugo-v0.80.0"
```

Comment:
Why the downgrade?

Reply:
because it doesnt work (the upgrade was done without any testing)